Mathlib/SetTheory/Game/PGame.lean

```diff
@@ -1350,6 +1350,18 @@ theorem moveRight_neg_symm {x : PGame} (i) :
 theorem moveRight_neg_symm' {x : PGame} (i) :
     x.moveRight i = -(-x).moveLeft (toLeftMovesNeg i) := by simp
 
+theorem neg_identical_neg_iff : ∀ {x y : PGame.{u}}, x ≡ y ↔ -x ≡ -y
```

Comment:
```suggestion
@[simp] theorem neg_identical_neg_iff : ∀ {x y : PGame.{u}}, -x ≡ -y ↔ x ≡ y 
```
This one should be a simp lemma, in this direction. It's useful to add `Identical.neg` as an alias for the reverse direction, for dot notation